<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-ERF5AP4R" name="restassuredtestcases.Testcase1" tests="1" failures="0" timestamp="31 May 2021 08:12:44 GMT" time="0.430" errors="1">
  <testcase name="datasourceexample" time="0.430" classname="restassuredtestcases.Testcase1">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: datasourceexample([Parameter{index=0, type=java.lang.CharSequence, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=double, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=6, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.Double) 2345.0,(java.lang.String) Maha,(java.lang.String) Savings,(java.lang.Double) 9.938438584E9,(java.lang.String) 13-98,(java.lang.String) Vizag,(java.lang.String) Andhrapradesh]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: datasourceexample([Parameter{index=0, type=java.lang.CharSequence, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=double, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=6, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.Double) 2345.0,(java.lang.String) Maha,(java.lang.String) Savings,(java.lang.Double) 9.938438584E9,(java.lang.String) 13-98,(java.lang.String) Vizag,(java.lang.String) Andhrapradesh]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:45)
at org.testng.internal.Parameters.injectParameters(Parameters.java:796)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:983)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- datasourceexample -->
</testsuite> <!-- restassuredtestcases.Testcase1 -->
